.github/LABEL_MAP.md

```diff
@@ -0,0 +1,4 @@
+Bug Report: type:bug
+Feature Request: type:enhancement
+UI Component: component:ui
+API Component: component:api
```

Comment:
Does this need a `component:installer` section too?

Reply:
It does.